vsts/pipelines/ci-detector.yml

```diff
@@ -0,0 +1,94 @@
+resources:
+- repo: self
+
+variables:
+- group: Oryx
+
+stages:
+  - stage: ValidatePipeline
+    displayName: Validate Pipeline
+    jobs:
+    - job: ValidatePipeline
+      displayName: Validate pipeline
+      pool:
+        name: OryxLinux
+      variables:
+        skipComponentGovernanceDetection: true
+      steps:
+      - script: |
+          set -x
+          if [ "$(Build.Reason)" != "Manual" ]; then
```

Comment:
why only manual?

Reply:
My thought was if we want to keep releasing nuget packages if the source code hasn't changed at all. For example, this could happen in case of `scheduled` automated builds. We can kick off a CI build if we know that we made changes to Detector api. Also the process of upgrading a pre-release package to released package should be done consciously and manually.